website/settings/defaults.py

```diff
@@ -488,6 +489,7 @@ class CeleryConfig:
                   routing_key=task_med_queue, consumer_arguments={'x-priority': 1}),
             Queue(task_high_queue, Exchange(task_high_queue),
                   routing_key=task_high_queue, consumer_arguments={'x-priority': 10}),
+            Queue(account_status_changes, Exchange(account_status_changes), routing_key=account_status_changes)
```

Comment:
Make the `x-priority` something already on this list makes this hang, but only on CI